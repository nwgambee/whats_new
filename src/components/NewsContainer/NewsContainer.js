import React from 'react';
import './NewsContainer.css'
import NewsArticle from '../NewsArticle/NewsArticle.js'

// NEWSCONTAINER COMPONENT CODE GOES HERE

const NewsContainer = (props) => {
  console.log(props);
  let articles = props.currentNews.map(article => {
    return <NewsArticle
      headline={article.headline}
      img={article.img}
      description={article.description}
      url={article.url} />
  })
  return (
    <div className='NewsContainer'>
      {articles}
    </div>
  )
}

export default NewsContainer;
