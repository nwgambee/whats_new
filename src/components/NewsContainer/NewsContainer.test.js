import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import NewsContainer from './NewsContainer';

describe('NewsContainer', () => {
  it('should match the snapshot', () => {
    const wrapper = shallow(<NewsContainer
      currentNews = {[1,2,3]}
          />);

    expect(wrapper).toMatchSnapshot();
  });
});
