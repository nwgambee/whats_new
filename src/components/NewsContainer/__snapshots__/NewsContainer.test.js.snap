// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewsContainer should match the snapshot 1`] = `
<div
  className="NewsContainer"
>
  <NewsArticle />
  <NewsArticle />
  <NewsArticle />
</div>
`;
